### DISCUSSION TITLE
How to retrieve memberslingbox details via Postman

### DISCUSSION THREAD

**Discussion by abarberenaCPDS** [GerryDazoo/Slinger#400] [General]
### Discussed in https://github.com/GerryDazoo/Slinger/discussions/389

<div type='discussions-op-text'>

<sup>Originally posted by **rbf121** November 30, 2022</sup>
As of 11/30/2022, the Sling APIs to retrieve your memberslingbox details are still online.  You can't get them from the original URL we were using but the underlying APIs are still active.

You need to call 2 APIs in order to get the memberslingbox details.  

You can use Postman to create the 2 required requests.

**Request 1:**

POST https://secure.sling.com/slingaccounts/rest/v3/identitypoint?contentType=application/json

Under Body select x-www-form-urlencoded
Enter following key / values

| key | value |
| - | - |
| context | SLINGIT_PORTAL |
| policy_code | MAS |
| email_address | {user-email} |
| password | {user-password} |

Hit send.  If it worked, you should get a 200 OK with JSON response.  Save the identity and ticket from response, you will need those for the next call.

**Request 2:**

New Request
GET https://secure.sling.com/slingaccounts/rest/v5/member/{member-id}/device/slingboxes?contentType=application/json

Replace {member-id} with the "identity" value from first request

Under Authorization tab, select Basic Auth.
Enter the "ticket" value from first request as the username.  Leave the password blank.
Hit Send.

It should list out your memberslingbox JSON.

Thanks @socelillyhammer for pointing out that the slingplayer desktop auth was still working.  Hopefully this is helpful for those that need to recover their account box info and/or can't install slingplayer desktop for whatever reason. 
</div>

**Comment by abarberenaCPDS** [comment]
Just confirmed, will update the documentation 📝

**Comment by panoriega** [comment]
I'm not on US, so I had to use VPN to make it work. Thanks

**Comment by davepins0n** [comment]
I tried the first curl command and got "OpenSSL SSL_read: Connection reset by peer, errno 104" What did I do wrong?

 curl --location --request POST 'https://secure.sling.com/slingaccounts/rest/v3/identitypoint' --header 'Content-Type: application x-www-form-urlencoded' --data-urlencode 'context=SLINGIT_PORTAL' --data-urlencode 'policy_code=MAS' --data-urlencode 'email_address=<my email>' --data-urlencode 'password=<my passwd>'

email and password have been omitted on purpose.

**Reply by GerryDazoo** [reply]
you need an email-address password to login into Sling. Or did you leave it out of your post on purpose?

**Reply by davepins0n** [reply]
Omitted on purpose

**Reply by GerryDazoo** [reply]
If your on Windows try replacing those single quotes with double quotes. Next time please post the command and all program output.

**Reply by davepins0n** [reply]
That worked... thanks

**Reply by matthewjt** [reply]
I know this is an old thread, but I can't seem to remember my password.  Anyone know if there's a method for getting it reset or getting the admin password without thee service?
